sunpy/image/resample.py

```diff
@@ -26,7 +26,7 @@ def resample(orig, dimensions, method='linear', center=False, minusone=False):
         Original input array.
     dimensions : `tuple`
         Dimensions that new `numpy.ndarray` should have.
-    method : {'neighbor' | 'nearest' | 'linear' | 'spline'}, optional
+    method : ``{'neighbor' | 'nearest' | 'linear' | 'spline'}``, optional
```

Comment:
Wasn't sure if this was the best idea.

Reply:
Seems good to me

Reply:
I wonder if `method : {``'neighbor'`` | ``'nearest'`` | ``'linear'`` | ``'spline'``}, optional`
or in fact
`method : {"neighbor" | "nearest" | "linear" | "spline"}, optional`

Reply:
there's proper numpydoc syntax for this and I don't think this is it.

Reply:
![image](https://user-images.githubusercontent.com/1391051/123137204-a1f8b980-d44b-11eb-9df9-803ca263421b.png)

https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard